tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+envlist = py3, pycodestyle, pyflakes, pylint
+recreate = True
+
+[testenv]
+basepython = python3.6
```

Comment:
Callingout 3.6 specifically prevents testing on Xenial. Might not be a big deal, but sooner or later Cosmic or later will see 3.7 and this will fail without 3.6 specifically not being installed.

Reply:
ack. fixed.